src/esp/gfx/Simulator.h

```diff
@@ -88,7 +88,7 @@ class Simulator {
   int getPhysicsObjectLibrarySize();
 
   // remove object objectID instance in sceneID
-  void removeObject(const int objectID, const int sceneID = 0);
+  int removeObject(const int objectID, const int sceneID = 0);
```

Comment:
Maybe document what the return value is? It wasn't immediately clear to me :)

Reply:
Good point, I'll do a documentation pass over some of these Simulator functions once the first physics doc PR is merged and link to relevant functions.

Reply:
Addressed with #211 as you commented there :)